nemo_megatron_launcher/README.md

```diff
@@ -0,0 +1,4808 @@
+# NeMo Megatron
+## Open Beta
+
+Scripts and code to provide end-to-end data preparation and training for
+NeMo-Megatron.
+
+The most recent version of the README can be found at [https://ngc.nvidia.com/containers/ea-bignlp:bignlp-training](https://ngc.nvidia.com/containers/ea-bignlp:bignlp-training).
```

Comment:
This is no longer true right? This file should be the most recent version?

Reply:
hmm, not sure how to change this now, but you are right. It should be updated to github.